### PR TITLE
Grunt lint and syntax fixes

### PR DESCRIPTION
**Pull request by jadekler**
- Consistent spacing for class functions in examples/
- Consistency for render(), spaced to avoid line overflow in `basic-click-counter/index.html`
- Fixed grunt lint errors in `ResponderTouchHistoryStore.js`, `ReactEmptyComponent-test.js`

### CODE REVIEW

File: examples/transitions/index.html [hunk 1]
```diff
@@ -35,19 +35,15 @@
         getInitialState: function() {
           return {current: 0};
         },
-
```
Comment (zpao) [inline examples/transitions/index.html]: I actually prefer the whitespace - code needs to breathe. Let's leave this.

File: src/renderers/shared/reconciler/__tests__/ReactEmptyComponent-test.js [hunk 2]
```diff
@@ -135,46 +135,45 @@ describe('ReactEmptyComponent', function() {
   );
 
   it('should have getDOMNode return null when multiple layers of composite ' +
-    'components render to the same null placeholder', () => {
-      spyOn(console, 'log');
+  'components render to the same null placeholder', () => {
```
Comment (zpao) [inline src/renderers/shared/reconciler/__tests__/ReactEmptyComponent-test.js]: Can you leave this line indented please?

File: examples/transitions/index.html (hunk 1 above)
Comment (jadekler) [inline examples/transitions/index.html]: Cool, I also do. I noticed the other examples (see examples/jquery-bootstrap: https://github.com/facebook/react/blob/master/examples/jquery-bootstrap/js/app.js#L19-L31) smush functions together. Would you like me to make these other examples consistent with the whitespaced functions, or just keep the mix?

File: src/renderers/shared/reconciler/__tests__/ReactEmptyComponent-test.js (hunk 2 above)
Comment (jadekler) [inline src/renderers/shared/reconciler/__tests__/ReactEmptyComponent-test.js]: On it.

**Comment by jadekler** [comment]
@zpao Do you guys prefer to create a new merge request, force push on top of the current merge request, add commits on top of the current merge request, or some other method?

File: examples/transitions/index.html (hunk 1 above)
Comment (zpao) [inline examples/transitions/index.html]: Let's change the other ones :)

**Comment by zpao** [comment]
Generally squash and force-push is preferred but we're not consistently sticklers.

File: examples/transitions/index.html (hunk 1 above)
Comment (jadekler) [inline examples/transitions/index.html]: Sweet, get you that change in a moment.

**Comment by jadekler** [comment]
@zpao Updated and squashed two commits + today's changes into one commit. Let me know if anything else isn't looking kosher.

**Comment by jadekler** [comment]
Huh. CI failed on [lint](https://travis-ci.org/facebook/react/jobs/86522016) and told me to use 6 spaces, but locally lint tells me to use 4 spaces. Weird. Taking it back to 6 to see if CI is happy.

**Comment by zpao** [comment]
Local lint is right, I think CI dependency caching screws things up (or more likely since our dep is a `^`, it doesn't upgrade the cached version unless we explicitly bump our dep). I bet if I clear caches and re-run your build it fails. I'm going to do a couple linting changes in the next couple days so hopefully it locks down better after that. In the meantime, sorry for the churn :(

**Comment by zpao** [comment]
Hey @jadekler, sorry to do this to you but I merged another PR that also fixed the lint issues since it was updated and the failures on travis were annoying (that's what I get for clearing the cache 😢). I'd love to still take the other style changes if you rebase though.

**Comment by zpao** [comment]
It's been a couple weeks without a response so I'm going to close out. Feel free to reopen / make a new PR with the stylistic changes if you come back to this.

**Pull request closed without merging**